docs/aerodromes/classc/Goldy.md

```diff
@@ -206,6 +206,16 @@ Both hospital helipads are outside the maneuvering area and do not require a tak
 
 All other aircraft require a 'Next' call to CG TCU.
 
+!!! phraseology
+    <span class="hotline">**CG ADC** -> **BAC**</span>: "Next, CBN, runway 14"  
+    <span class="hotline">**BAC** -> **CG ADC**</span>: "CBN, heading 030, unrestricted"  
+    <span class="hotline">**CG ADC** -> **BAC**</span>: "Heading 030, CBN"  
+
+    **CG ADC**: "CBN, Assigned heading left 030, Runway 14, Cleared for Takeoff"  
+    **CBN**: "Left heading 030, Runway 14, Cleared for Takeoff, CBN"
+
+The BN TCU controller can suspend/resume Auto Release at any time, with the concurrence of **CG ADC**.
+
```

Comment:
Might just be a merge thing here, but I removed these location-specific examples and instead provided better examples on the linked controller skills page for Next coord. @alphadelta332 are you happy if we remove this?